### ISSUE TITLE
Open from local file

### ISSUE THREAD

**Issue by K2ouMais** [JustSteveKing/laravel-stoplight-elements#2]
Hey,

is there a way to use your package with a local file?

I tried to use resource_path('apidoc/openapi.yaml') in your config and it seems that is doesnt work.

![404](https://user-images.githubusercontent.com/52510455/118638414-1c904280-b7d7-11eb-94dc-344af935bd8b.png)

If I let your package on default, the default documentation looks like this:

![external](https://user-images.githubusercontent.com/52510455/118638498-3467c680-b7d7-11eb-9f30-91f1c486129b.png)

Other than this... Thank you for your take on this, as I love Stoplight to design my APIs and only need a way to show them on my site.

**Comment by JustSteveKing** [comment]
This is unplug CDN returning a 404 for the style script, I have passed this onto Stoplight support now, and should receive an answer soon, so I can either publish an update or find a fix.

I noticed on twitter you said you were using PHP 7, I will push an update to allow PHP 7 also, as there isn't too much going on that "needs" PHP8 - it is just a general practice I follow

**Comment by JustSteveKing** [comment]
@K2ouMais Also, having just tested - the path to the file needs to be a URL that can be reached by the browser. I would recommend:

```php
Route::get('openapi', function () {
    return response()->file(config('stoplight.openapi.path));
})->name('openapi');
```

Then in your ENV you can add the correct path, or override the configuration.

**Comment by K2ouMais** [comment]
Are you getting the same CDN problem? I am asking because I wanted to test your package at my working place and the problem could be on my side and eventually I have to ask our administrators to open the firewall for the CDN.

Yes, if there isnt a problem and you havent used the PHP 8 syntax, it would be really helpfull to tag a release with PHP 7 support. 

Thank you for all the effort.

**Comment by JustSteveKing** [comment]
I have now published a new release, version 0.0.3 fixes the styling issue thanks to @philsturgeon and version 0.0.4 adds PHP 7.4 support as we are not using any PHP 8 features in any way.

As per the original issue of using a local file, you will need to serve this document as it needs to be accessible over HTTP I believe - at least that is what I found in my testing.

I am closing this issue for now, however please feel free to report back if you have the same issue, and I shall reopen and investigate more.

**Comment by K2ouMais** [comment]
Thank you for supporting PHP 7.4... I think there will be another release. I reported some bugs to the stoplight elements repository and these were already merged.
